### PR TITLE
Add EventTarget compat data

### PR DESCRIPTION
**Pull request by chrisdavidmills**
I was documenting the new constructor anyway, so I thought "might as well" ;-)

### CODE REVIEW

File: api/EventTarget.json [hunk 1]
```diff
@@ -0,0 +1,507 @@
...
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "1"
+            },
+            "firefox_android": {
+              "version_added": "1"
+            },
+            "ie": {
+              "version_added": "9",
```
Comment (Elchi3) [inline api/EventTarget.json]: I would write this as

``` 
"ie": [
  {
    version_added: "9"
  },
  {
    "alternative_name": "attachEvent",
    "version_added": "true",
    "notes": "Older versions of IE supported an equivalent, proprietary, <code>EventTarget.attachEvent()</code> method."
  }
],

File: api/EventTarget.json [hunk 2]
```diff
@@ -0,0 +1,507 @@
...
+              }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "options": {
```
Comment (Elchi3) [inline api/EventTarget.json]: woah, this is rather complex. I think I would have put several sub features here. (It's fine to nest deeper)
- options (desc: "Form with <code>options</code> object supported (third parameter can be either options or a <code>Boolean</code>, for backwards compatibility)
  - capture (desc: "`capture` option")
  - once (desc: "`once` option")
  - passive (desc: "`passive` option")
  - passive_true_touch (desc: " <code>passive</code> option defaults to true for <code>touchstart</code> and <code>touchmove</code> events"

File: api/EventTarget.json [hunk 3]
```diff
@@ -0,0 +1,507 @@
...
+        }
+      },
+      "addEventListener": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/EventTarget/addEventListener",
+          "support": {
+            "webview_android": {
+              "version_added": "1"
+            },
+            "chrome": {
```
Comment (Elchi3) [inline api/EventTarget.json]: Chrome has a note here: "Before Chrome 49, the `type` and `listener` parameters were optional."

File: api/EventTarget.json [hunk 4]
```diff
@@ -0,0 +1,507 @@
...
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "2"
+            },
+            "firefox_android": {
+              "version_added": "2"
+            },
+            "ie": {
```
Comment (Elchi3) [inline api/EventTarget.json]: I would write this as

``` 
"ie": [
  {
    version_added: "9"
  },
  {
    "alternative_name": "fireEvent",
    "version_added": "true",
    "notes": "Older versions of IE supported an equivalent, proprietary, <code>EventTarget.fireEvent()</code> method."
  }
],

File: api/EventTarget.json [hunk 5]
```diff
@@ -0,0 +1,507 @@
...
+              "version_added": "9",
+              "notes": "Older versions of IE supported an equivalent, proprietary, <code>EventTarget.fireEvent()</code> method."
+            },
+            "opera": {
+              "version_added": "51"
```
Comment (Elchi3) [inline api/EventTarget.json]: 9

File: api/EventTarget.json [hunk 6]
```diff
@@ -0,0 +1,507 @@
...
+            },
+            "opera_android": {
+              "version_added": "51"
```
Comment (Elchi3) [inline api/EventTarget.json]: true

File: api/EventTarget.json [hunk 7]
```diff
@@ -0,0 +1,507 @@
...
+            },
+            "safari": {
+              "version_added": false
```
Comment (Elchi3) [inline api/EventTarget.json]: 3.2

File: api/EventTarget.json [hunk 8]
```diff
@@ -0,0 +1,507 @@
...
+            },
+            "safari_ios": {
+              "version_added": false
```
Comment (Elchi3) [inline api/EventTarget.json]: true

File: api/EventTarget.json [hunk 9]
```diff
@@ -0,0 +1,507 @@
...
+              "version_added": "4"
+            },
+            "chrome": {
+              "version_added": "4"
+            },
+            "chrome_android": {
+              "version_added": "4"
+            },
+            "edge": {
+              "version_added": null
```
Comment (Elchi3) [inline api/EventTarget.json]: true

File: api/EventTarget.json [hunk 10]
```diff
@@ -0,0 +1,507 @@
...
+            },
+            "edge_mobile": {
```
Comment (Elchi3) [inline api/EventTarget.json]: true

File: api/EventTarget.json [hunk 11]
```diff
@@ -0,0 +1,507 @@
...
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "removeEventListener": {
```
Comment (Elchi3) [inline api/EventTarget.json]: I would mirror the table of `addEventListener` here. See my comments above.

File: api/EventTarget.json (hunk 3 above)
Comment (chrisdavidmills) [reply api/EventTarget.json]: Added

File: api/EventTarget.json (hunk 1 above)
Comment (chrisdavidmills) [reply api/EventTarget.json]: OK, done

File: api/EventTarget.json (hunk 2 above)
Comment (chrisdavidmills) [reply api/EventTarget.json]: Ah, this makes sense. Done - let me know if it's right ;-)

File: api/EventTarget.json (hunk 9 above)
Comment (chrisdavidmills) [reply api/EventTarget.json]: changed

File: api/EventTarget.json (hunk 10 above)
Comment (chrisdavidmills) [reply api/EventTarget.json]: changed.

File: api/EventTarget.json (hunk 4 above)
Comment (chrisdavidmills) [reply api/EventTarget.json]: Done

File: api/EventTarget.json (hunk 5 above)
Comment (chrisdavidmills) [reply api/EventTarget.json]: changed

File: api/EventTarget.json (hunk 6 above)
Comment (chrisdavidmills) [reply api/EventTarget.json]: changed

File: api/EventTarget.json (hunk 7 above)
Comment (chrisdavidmills) [reply api/EventTarget.json]: changed

File: api/EventTarget.json (hunk 8 above)
Comment (chrisdavidmills) [reply api/EventTarget.json]: changed

File: api/EventTarget.json (hunk 11 above)
Comment (chrisdavidmills) [reply api/EventTarget.json]: According to the removeEventListener page, that complexity isn't needed; the support info is simpler. So I've not made quite as many changes here.

File: api/EventTarget.json [hunk 12]
```diff
@@ -0,0 +1,711 @@
...
+              "description": "<code>capture</code> option",
+              "support": {
+                "webview_android": {
+                  "version_added": true
+                },
+                "chrome": {
+                  "version_added": true
+                },
+                "chrome_android": {
+                  "version_added": true
```
Comment (jpmedley) [inline api/EventTarget.json]: 52 for all 3 flavors of Chrome.

**Pull request merged**